app/controllers/reservations_controller.rb

```diff
@@ -183,7 +183,24 @@ def update
       return
     end
 
-    @reservation.assign_times_from_params(params[:reservation])
+    reservation_params = params[:reservation]
```

Comment:
What if we took this entire chunk and extracted it into a `reservation_params` method?